cogs/TextFilter.py

```diff
@@ -0,0 +1,701 @@
+#!/usr/bin/env python
+
+"""
+Koala Bot Text Filter Code
+"""
+
+# Built-in/Generic Imports
+import os
+import asyncio
+import time
+import re
+import logging
+
+# Libs
+from discord.ext import commands, tasks
+import discord
+
+# Own modules
+import KoalaBot
+from utils.KoalaColours import *
+from utils.KoalaUtils import error_embed, is_channel_in_guild, extract_id
+from utils.KoalaDBManager import KoalaDBManager
+
+
+def text_filter_is_enabled(ctx):
+    """
+    A command used to check if the guild has enabled TextFilter
+    e.g. @commands.check(KoalaBot.is_admin)
+    :param ctx: The context of the message
+    :return: True if admin or test, False otherwise
+    """
```

Comment:
Odd, but quick edit with all the docstrings. We've had an issue in the past with Sphinx not noticing these docstrings correctly, adding a return before parameters seems to fix it
```suggestion
    """
    A command used to check if the guild has enabled TextFilter
    e.g. @commands.check(KoalaBot.is_admin)
    
    :param ctx: The context of the message
    :return: True if admin or test, False otherwise
    """
```